root.sh

```diff
@@ -60,6 +60,7 @@ fi
 
 # Disable Python for non-ROOT 6 builds
 [[ -d $SOURCEDIR/interpreter/llvm ]] || NO_PYTHON=1
+[[ $(python  'import sys; sys.exit(0 if sys.version_info[0] > 2 else 1)') -eq 0 ]] && [[ ! $NO_PYTHON ]] && IS_PYTHON3=1
```

Comment:
This line is wrong, I believe you meant to write:

```bash
[[ $NO_PYTHON ]] || { [[ $(python 'import sys; print(sys.version_info[0])') < 3 ]] || IS_PYTHON3=1; }
```

Which means:

* Check if we need to _disable_ Python. In case we don't, check Python major version.
* Check if the major version is below 3. In case it isn't (Python 3 case) then set `IS_PYTHON3` to `1`.